Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -1,17 +1,17 @@
 """Test file for Illumio Integration."""
 
 import io
-import json
+
+import illumio.pce
 import pytest
-from illumio import PolicyComputeEngine, IllumioException
-from CommonServerPython import *  # noqa
-from IllumioCore import InvalidMultiSelectException, InvalidSingleSelectException, InvalidPortException, \
-    VALID_POLICY_DECISIONS, VALID_PROTOCOLS
 
+from CommonServerPython import *  # noqa
+from IllumioCore import *
+from test_data.test_data_responses import *
 
-'''CONSTANTS'''
+""" CONSTANTS """
 
-VIRTUAL_SERVICE_URL = "https://127.0.0.1:8443/api/v2/orgs/1/sec_policy/draft/virtual_services"
+DIRECTORY_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test_data")
```

Comment:
Rename this `TEST_DATA_DIRECTORY` or similar, DIRECTORY_PATH is too generic.

Reply:
Okay